eng/testing/tests.mobile.targets

```diff
@@ -38,6 +38,9 @@
     <HttpActivityPropagationSupport Condition="'$(HttpActivityPropagationSupport)' == ''">false</HttpActivityPropagationSupport>
     <UseSystemResourceKeys Condition="'$(UseSystemResourceKeys)' == ''">false</UseSystemResourceKeys>
     <UseNativeHttpHandler Condition="'$(UseNativeHttpHandler)' == ''">false</UseNativeHttpHandler>
+    <DynamicCodeSupport Condition="'$(DynamicCodeSupport)' == '' and '$(MonoForceInterpreter)' != 'true'">false</DynamicCodeSupport>
+    <_DefaultValueAttributeSupport Condition="'$(_DefaultValueAttributeSupport)' == ''">true</_DefaultValueAttributeSupport>
+    <_DesignerHostSupport Condition="'$(_DesignerHostSupport)' == ''">true</_DesignerHostSupport>
```

Comment:
FWIW
I think these need to be in a separate property group as I believe `BuildTestsOnHelix=true` in our setup.
So based on my previous comment, I think we will need a new property group conditioned with `'$(EnableAggressiveTrimming)' == 'true' and '$(BuildTestsOnHelix)' == 'true'` and then add these switches there.

Reply:
Thanks, I missed the condition.